### DISCUSSION TITLE
Avoiding css classes header

### DISCUSSION THREAD

**Discussion by RannyBergamotte** [chuangcaleb/obsidian-fountain-editor#32] [Ideas]
Hi! new to Obsidian, but this really caught my attention, and wanted to check it out and learn more about obsidian. My only pet peeve is coming from Better Fountain for vscode, that there is the
```md
---
cssclasses:
  - fountain
---
```
at the beginning. When exporting the screenplay into a pdf, it messes with the export and I have to remove it before hand. Maybe stupid idea, but I was trying to be able to make the vault location and vs-code better fountain folder location the same, so I can easily switch between the two when I wanted to (I think both are super powerful in different ways, and synergize really well together). It would be awesome if it could automatically over-ride if it detects it to be a .fountain file instead of .md.

**Comment by chuangcaleb** [comment]
No no, co-locating the Obsidian vault and VS Code fountain folder location (repo) is exactly intended in the design!

You can use [Obsidian Custom File Extensions Plugin](obsidian://show-plugin?id=obsidian-custom-file-extensions-plugin) to register "fountain" as markdown. This allows you to edit and work inside a `.fountain` file as if it were a `.md` file, and get all the same markdown syntax. The plugin already detects the `.fountain` extension and will apply fountain formatting accordingly. See below for the exact JSON config.

<details><summary>Custom File Extensions Config</summary>
<p>
<pre>
{
  "markdown": [
    "",
    "md",
    "txt",
    "js",
    "css",
    "ts",
    "jsx",
    "tsx",
    "yaml",
    "yml",
    "sass",
    "scss",
    "tex",
    "json",
    "html",
    "fountain"
  ]
}
</pre>

</p>
</details> 

Otherwise, you can just use the Obsidian Longform plugin to compile your selected markdown file segments, stripping the metadata headers, and then saving to a `output.fountain` file — then open that `.fountain` file with VS Code to export with Better Fountain.

Let me know any further questions!
